editor/src/components/canvas/canvas-strategies/absolute-resize-bounding-box-strategy.tsx

```diff
@@ -36,29 +36,17 @@ import {
 } from './shared-absolute-resize-strategy-helpers'
 import * as EP from '../../../core/shared/element-path'
 import { ZeroSizeResizeControlWrapper } from '../controls/zero-sized-element-controls'
+import { SetCssLengthProperty, setCssLengthProperty } from '../commands/set-css-length-command'
 
 export const absoluteResizeBoundingBoxStrategy: CanvasStrategy = {
   id: 'ABSOLUTE_RESIZE_BOUNDING_BOX',
   name: 'Absolute Resize',
   isApplicable: (canvasState, interactionState, metadata, allElementProps) => {
     const filteredSelectedElements = getDragTargets(canvasState.selectedElements)
-    if (
-      filteredSelectedElements.length > 1 ||
-      (filteredSelectedElements.length >= 1 &&
-        (interactionState?.interactionData.modifiers.alt ||
-          interactionState?.interactionData.modifiers.shift))
-    ) {
-      return canvasState.selectedElements.every((element) => {
-        const elementMetadata = MetadataUtils.findElementByElementPath(metadata, element)
-        const elementProps = allElementProps[EP.toString(element)] ?? {}
-        return (
-          elementMetadata?.specialSizeMeasurements.position === 'absolute' &&
-          hasAtLeastTwoPinsPerSide(elementProps) // TODO should this use projectContents?
-        )
-      })
-    } else {
-      return false
-    }
+    return filteredSelectedElements.every((element) => {
```

Comment:
Shouldn't we still check that the number of selected elements is at least 1? I guess without that there is no control to drag though, so I'm unsure if it is necessary or not, but we still check that for other strategies.

Reply:
fixed